atd-vzd/triggers/atd_txdot_crashes_updates_audit_log.sql

```diff
@@ -152,15 +152,6 @@ BEGIN
     NEW.atd_mode_category_metadata = get_crash_modes(NEW.crash_id);
     --- END OF MODE CATEGORY DATA ---
 
-    ------------------------------------------------------------------------------------------
-    -- AUSTIN FULL PURPOSE
-    ------------------------------------------------------------------------------------------
-    -- Set Austin Full Purpose to Y (TRUE) when it has Austin City ID and no coordinates.
-    IF (NEW.position IS NULL and NEW.city_id = 22) THEN
-        NEW.austin_full_purpose = 'Y';
-    END IF;    
-    --- END OF AUSTIN FULL PURPOSE ---
-
```

Comment:
dont need this anymore, will be handled in new `in_austin_full_purpose` generated column